pyproject.toml

```diff
@@ -23,13 +23,13 @@ click = "^7.0"
 pyparsing = "^2.4.6"
 python = "^3.6"
 pyyaml = "^5.3"
-receptor = { git = "https://github.com/project-receptor/receptor.git" }
 requests = "^2.22.0"
 setuptools = "*"  # For pkg_resources
 wait-for = "^1.1.1"
 python-dateutil = "^2.8.1"
 prometheus-client = "^0.7.1"
 psutil = "^5.6.7"
+receptor = {git = "https://github.com/project-receptor/receptor.git", rev = "devel"}
```

Comment:
Shouldn't this use the "branch" keyword, not the "rev" keyword? See: https://python-poetry.org/docs/dependency-specification/#git-dependencies

Reply:
yep! updated

Reply:
updated and also reran `poetry lock` again just in case